.buildkite/scripts/steps/checks/no_codeowners.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+if [[ "${GITHUB_PR_LABELS:-}" == *"backport"* ]]; then
```

Comment:
This is going to match anything that includes `backport`, such as `auto-backport`

Reply:
Ugh, good point. Originally I was checking for `GITHUB_PR_TARGET_BRANCH != "main"` but that wouldn't work for PRs targeting feature branches. Do you think there's a chance we can do an inclusion check for labels rather than globby-string matches? I know there's some sort of array type in bash

Reply:
Maybe we could do something with the new versions.json file to determine that this is specifically a PR to a backport target?

Reply:
`[[ ","${GITHUB_PR_LABELS:-}"," == *",backport,"* ]]`

thanks StackOverflow

Reply:
Ugh, making our crazy string matching even crazier is something I'd love to avoid.

Reply:
@brianseeders looks like we could do something like:

```
jq ".versions[] | select(.branch | contains(${GITHUB_PR_TARGET_BRANCH:BUILDKITE_BRANCH}))" versions.json
```

To determine if we're running on a tracked branch, so combine that with `GITHUB_PR_TARGET_BRANCH != "main"` and we could probably run this on-merge and in PRs

Reply:
You could do `[[ "$GITHUB_PR_TARGET_BRANCH" != "main" && "$BUILDKITE_PULL_REQUEST$BUILDKITE_BRANCH" != "main" ]]` and it will run everywhere except `main` and PRs targeting `main`

Reply:
Yeah, but then it would also run on PRs targeting feature branches which will end up in main